src/core/agent/services/connectionService.ts

```diff
@@ -80,27 +97,28 @@ class ConnectionService extends AgentService {
         isMultiSigInvite: multiSigInvite,
         connectionId: undefined,
         status: ConnectionStatus.PENDING,
+        url,
       },
     });
 
-    const operation = await this.resolveOobi(url, multiSigInvite);
```

Comment:
We still need to call resolveOobi if it's for multisig.. please see AC 5 and test this properly for multisig

Reply:
We were calling `resolveOobi()` by event emitter for both cases immediately after creating the event `ConnectionStatus.PENDING` in function`connectByOobiUrl`

Reply:
No, for multi-sig, we need to call resolveOobi and wait for the operation to complete for a specific amount of time and throw an error if not. It's a different flow.

Only track the long running operation in the DB with events if it's a normal connection.